gdal-sys/src/ogr_enums.rs

```diff
@@ -36,3 +36,6 @@ pub enum OGRFieldType {
     OFTInteger64List = 13,
     OFTMaxType = 14
 }
+
+pub const FALSE: i32 = 0;
+pub const TRUE: i32 = 1;

```

Comment:
Is the `i32 `always correct or is this a `c_int`?
Maybe we cloud call it `C_TRUE `and `C_FALSE` ?

Reply:
Good point. Bikeshed: maybe `OGR_TRUE`?

Reply:
well it's not really a OGR constant... So i would prefer the C prefix.

Reply:
`CPL_` would have been the best :smile:

Reply:
GDAL prefixes are a bit confusing sometimes 😄